server/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const app = express()
+const sqlite = require('sqlite3').verbose()
+const compression = require('compression')
+
+const PORT = 5000
+
+app.use(express.static('server/static'))
+app.use(compression())
+
+app.get('/api/v1/trees', (req, res) => {
+  let db = new sqlite.Database('./database.sqlite3')
```

Comment:
If we stick with node, then we'll surely end up with more endpoints. With that in mind, consider moving the database connection its own separate function.

```suggestion
function connection() {
  return (new sqlite.Database('./database.sqlite3'));
}
```

Reply:
Hmm, good point there. That is the method recommended in the docs, but it isn't a huge deal since we only have a few endpoints.